tools/shared.py

```diff
@@ -1244,41 +1244,50 @@ class Building:
   @staticmethod
   def get_multiprocessing_pool():
     if not Building.multiprocessing_pool:
-      cores = int(os.environ.get('EMCC_CORES') or multiprocessing.cpu_count())
-
-      # If running with one core only, create a mock instance of a pool that does not
-      # actually spawn any new subprocesses. Very useful for internal debugging.
-      if cores == 1:
-        class FakeMultiprocessor:
-          def map(self, func, tasks):
-            results = []
-            for t in tasks:
-              results += [func(t)]
-            return results
-        Building.multiprocessing_pool = FakeMultiprocessor()
-      else:
-        child_env = [
+      pool = Building.get_new_multiprocessing_pool()
+      Building.multiprocessing_pool = pool
+    return Building.multiprocessing_pool
+
+  @staticmethod
+  def get_new_multiprocessing_pool():
```

Comment:
Naming nit: `get` and `new` are different verbs. Maybe `create_multiprocessing_pool()`?